resources/templates/ebscoKbTitleList.php

```diff
@@ -0,0 +1,63 @@
+<table id='resource_table' class='dataTable table-striped' style='width:840px'>
+    <thead>
+        <tr>
+            <th><?php echo _("Title"); ?></th>
+            <th><?php echo _("Imported?"); ?></th>
+            <th><?php echo _("Resource Type"); ?></th>
+            <th><?php echo _("ISXNs"); ?></th>
+            <th><?php echo _("Import"); ?></th>
+        </tr>
+    </thead>
+    <tbody>
+    <?php foreach($items as $item): ?>
+        <?php $item->loadResource(); ?>
+        <tr>
+            <td>
+                <a
+                    href="ajax_htmldata.php?action=getEbscoKbTitleDetails&height=700&width=730&modal=true&titleId=<?php echo $item->titleId; ?>"
+                    class="thickbox">
+                    <?php echo $item->titleName; ?>
+                </a>
+            </td>
+            <td style="text-align: center;">
+                <?php if($item->resource): ?>
+                    <a href="resource.php?resourceID=<?php echo $item->resource->primaryKey; ?>">
+                        <i class="fa fa-check text-success" title="imported in Coral"></i>
+                    </a>
+                <?php endif; ?>
+            </td>
+            <td>
+                <?php echo $item->pubType; ?>
+            </td>
+            <td>
+                <ul style="list-style: none; font-size: .8em">
+                    <?php
+                    foreach($item->identifiersList as $identifier){
```

Comment:
I get the message
> PHP Warning:  Invalid argument supplied for foreach()

repeatedly when doing a search.

Reply:
I'm not seeing this error. Is there a particular search that's causing it or any search?

Also what version of php are you using? Just so I can make my dev environment the same as yours. Worse case scenario, I can throw a guard around the foreach loop, but I wanted to make sure it's not an issue with the model's getter functions.

Reply:
Environment:
>Server version: Apache/2.4.18 (Ubuntu)
mysql Ver 14.14 Distrib 5.7.18, for Linux (x86_64) using EditLine wrapper
PHP 7.1.6-1~ubuntu16.04.1+deb.sury.org+1 (cli) (built: Jun 9 2017 08:26:34) ( NTS )

I did the same "nursing" title search as you, but we return many books which seem to be the problem. See screenshot: https://screenshots.firefox.com/TSdNqbqc0oTO5UuS/coral.vm

…And I was just informed that these come from a custom package of ours that has potentially unpredictable data in some fields. We are turning that off in our EDS, and I will do the search again and report back if it all goes away, but maybe this is a potential problem with custom packages that needs to be known about.

Reply:
I see. Let me check the API and see if there is a way to distinguish between custom and managed packages. Otherwise, I can add some guards on those attributes to return empty arrays

Reply:
just pushed a fix. Let me know if it's not working.